pulse/service/src/main/java/gov/ca/emsa/pulse/service/PatientSearchTermsException.java

```diff
@@ -3,5 +3,5 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "One of the following fields were blank: DOB, Family/Given Name, Gender")
+@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "One of the following fields were blank or incorrect: DOB, Family/Given Name, Gender")
```

Comment:
As much as I hate it, the end users want "First & Last" when we talk about names. I think this error should read: "One of the following search parameters was blank or improperly formed: Name, Date of Birth, Gender"